llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Passes/TileAndDecomposeAttention.cpp

```diff
@@ -74,21 +74,38 @@ static Value computePartialSoftmax(Value qkTranspose, Value currentMax,
   return genericOp.getResult(0);
 }
 
-static Value updateAndScale(Value oldMax, Value newMax, Value oldSum,
-                            Location loc, OpBuilder &builder,
+static Value computeScaleFactor(Value oldMax, Value newMax, Location loc,
```

Comment:
nit: Add a brief comment like the following because it is mutating an argument.
```
// Return the scale factor for the new softmax maximum and add the generic to the provided list of ops.
```